### PR TITLE
Add Alliant Credit Union

### PR DESCRIPTION
**Pull request by kishoresurana**

### CODE REVIEW

**Comment by stephengroat** [comment]
Please add an img to `img/banking`, like this one:
alliantcreditunion

File: _data/banking.yml [hunk 1]
```diff
@@ -1,647 +1,654 @@
...
+      url: http://www.actorsfcu.com/
+      twitter: ActorsFCU
+      img: actorsfcu.png
+      tfa: No
+
+    - name: Alliant Credit Union
+      url: www.alliantcreditunion.com
+      twitter: AlliantCU
+      facebook: alliantcreditunion
+      img: actorsfcu.png
```
Comment (stephengroat) [inline _data/banking.yml]: Please use the proper image

**Review by stephengroat — DISMISSED** [review]
Please fix the image issue, `actorsfcu.png` is not the right image

**Comment by stephengroat** [comment]
@kishoresurana could you update the `img` tag and add the image?

**Comment by Jawshy** [comment]
@kishoresurana, there appears to be a merge conflict. Would you be able to resolve that as well as commit @stephengroat's [requested change](https://github.com/2factorauth/twofactorauth/pull/2091#pullrequestreview-3415569)?

**Comment by stephengroat** [comment]
@Jawshy think there's a line ending issue

**Comment by stephengroat** [comment]
Sorry, I meant like LF/CRLF issue. I think that's what you're seeing on the diff

**Comment by Carlgo11** [comment]
There you go

**Review by Jawshy — APPROVED** [review]
`looks-good`

**Pull request merged**